### PR TITLE
Development save in mongo+ update status fix+ expire check

### PR DESCRIPTION
**Pull request by ashishddgr8**
save in mongo+ update status fix+ expire check

### CODE REVIEW

**Comment by ashishddgr8** [comment]
[Offers Service: Save generated offers to MongoDB](https://trello.com/c/dmbdT2St/112-offers-service-save-generated-offers-to-mysql)

**Comment by ashishddgr8** [comment]
[Offers Service: Generate offer to optimize PartnerOrder](https://trello.com/c/xZGLfBns/111-offers-service-generate-offer-to-optimize-partnerorder)

File: src/main/java/com/incentives/piggyback/offers/controller/OfferController.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    		@RequestBody OfferDTO offer) {
+        log.debug("Offer Service: Received PUT request for updating offer");
+        return offerService.updateOfferStatus(offer);
+    }
+    
+    @PostMapping("/partner/order")
+    public ResponseEntity<OfferEntity> offerForPartnerOrder(@RequestBody OfferDTO offer) {
+        log.debug("Offer Service: Received POST request with offer data for partner {}", offer);
+        return offerService.offerForPartnerOrder(offer);
+    }
```
Comment (piggy1-mvn) [inline src/main/java/com/incentives/piggyback/offers/controller/OfferController.java]: There should be no Post/Put calls to Offers Service.

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -2,4 +2,7 @@ server.port = 8081
 spring.cloud.gcp.project-id=absolute-text-251105
 spring.cloud.gcp.credentials.location=file:/app/config/secret.json
 location.api.fetch.nearby.users=http://localhost:8080/location/user
-notification.api.broadcast=http://localhost:8086/notification/broadcast
\ No newline at end of file
+notification.api.broadcast=http://localhost:8086/notification/broadcast
+users.api.usersWithInterest=http://localhost:8083/userwithinterest
```
Comment (piggy1-mvn) [inline src/main/resources/application.properties]: We should always have /user/ as part of the API resource depicting which service we are calling, from the URL. If this resource has already been implemented on User service with this url, we should update User service as well.

**Pull request closed without merging**